frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditDate/EditDate.tsx

```diff
@@ -216,6 +276,50 @@ export const EditDate = ({ field }: EditDateProps): JSX.Element => {
           {get(errors, 'dateValidation.customMinDate.message')}
         </FormErrorMessage>
       </FormControl>
+      <Stack>
+        <FormControl isReadOnly={isLoading}>
+          <Toggle
+            {...register('addParticularDayRestriction')}
+            label="Customize days of the week"
+            description="Checking a day will disable all the same days in the calendar"
+          />
+        </FormControl>
+        {getValues('addParticularDayRestriction') ? (
+          <FormControl isRequired isReadOnly={isLoading}>
+            <Controller
+              control={control}
+              name="invalidDays"
+              render={({ field: { ref, ...field } }) => {
```

Comment:
`ref` seems unused here, but i suspect we want to put it in the first Checkbox (so errors will transfer focus to the correct component)